### PR TITLE
Update README.md to show Rails 4 support

### PR DESCRIPTION
**Pull request by ckib16**
I've suggested a minor change to clearly explain that Rails ERD now fully supports Rails 4. This update should make it clear to new users that they can fully utilize Rails ERD in their Rails 4 projects.

Thanks,
Chris

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Rails ERD - Generate Entity-Relationship Diagrams for Rails applications
 
 The second goal of Rails ERD is to provide you with a tool to inspect your application's domain model. If you don't like the default output, it is very easy to use the API to build your own diagrams.
 
-Rails ERD was created specifically for Rails 3. It uses Active Record's built-in reflection capabilities to figure out how your models are associated.
+Rails ERD was created specifically for Rails 3, but also supports Rails 4. It uses Active Record's built-in reflection capabilities to figure out how your models are associated.
```
Comment (bf4) [inline README.md]: why not just `Rails ERD was created specifically for Rails and works on versions 3.0-4.2` ?

Comment (ckib16) [inline README.md]: @bf4 Yes, I agree. That is probably better than what I wrote. Good call.

**Comment by bf4** [comment]
Looks fine to me. Would you mind rebasing?  http://help.exercism.io/git-workflow.html http://rakeroutes.com/blog/deliberate-git/

File: README.md (hunk 1 above)
Comment (ckib16) [inline README.md]: @bf4 I added another commit with your suggested wording. Let me know if there was anything non-standard in how I handled the modification to the original PR commit. This is my first PR to a public project, so I welcome any feedback! 

Take care,
Chris

**Comment by ckib16** [comment]
Will do. Let me tackle that tomorrow night. I have rebased before, but want to make sure I don't rush it late at night.

**Comment by ckib16** [comment]
Error in rebasing. Will open a new PR.

**Comment by bf4** [comment]
No need. I'll do it.

**Comment by ckib16** [comment]
Thanks for the offer @bf4. Looks like my new PR was accepted and submitted already. Appreciate the help on my 1st GitHub contribution.

**Comment by bf4** [comment]
No problem and congrats!  Let me know if you want some assistance in rebasing some time.

**Comment by ckib16** [comment]
@bf4 Sounds good, will do. Much appreciated.

**Comment by bf4** [comment]
Ping me at benjaminfleischer.com/pair

**Pull request closed without merging**